cli/pkg/kctrl/cmd/package/installed/ytt_overlay_flags.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package installed
+
+import "github.com/spf13/cobra"
+
+type YttOverlayFlags struct {
+	yttOverlays     bool
+	yttOverlayFiles []string
+}
+
+func (s *YttOverlayFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().StringSliceVar(&s.yttOverlayFiles, "ytt-overlay-file", nil, "A file or folder with ytt overlays to be applied to package install")
```

Comment:
I think we need to specify that it can be a folder too, but we can make this better, I agree

Reply:
> I think we need to specify that it can be a folder too

We usually don't do that, but if we want to, then how about `Path to ytt overlay file (can be a directory)`?

Reply:
Makes sense ! Adopted this